addons/binding/org.openhab.binding.icloud/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="icloud" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>iCloud Binding</name>
+    <description>Information about devices registered with iCloud.</description>
+    <author>Patrik Gfeller, Hans-Jörg Merk, Thomas Dietrich</author>

```

Comment:
We usually tend to leave out the author tag if it is a co-work; mentioning everyone that was involved will result in too long lists, so the idea is rather to only mention the main contributor/maintainer here or none.